### PR TITLE
change: removal of disabled flag from messages Refresh button

### PR DESCRIPTION
**Pull request by iarp**
In reference to #636

### CODE REVIEW

File: ui/src/message/Messages.tsx [hunk 1]
```diff
@@ -70,7 +70,6 @@ class Messages extends Component<IProps & Stores<'messagesStore' | 'appStore'>,
                         <Button
                             id="refresh-all"
                             variant="contained"
-                            disabled={!hasMessages}
```
Comment (jmattheis) [inline ui/src/message/Messages.tsx:73]: If the webapp is cache-reopened, then it's likely that the websocket connection isn't reconnected. So you will be able to refresh the messages, but this won't reconnect the websocket.

**Review by jmattheis — APPROVED** [review]
Thanks. It's weird, that this was disabled when there are no messages (:.

File: ui/src/message/Messages.tsx (hunk 1 above)
Comment (iarp) [reply ui/src/message/Messages.tsx:73]: Could we somehow get the refresh button to check connection status and call upon WebSocketStore.listen if needed? I'm not all familiar with how react works.

Comment (jmattheis) [reply ui/src/message/Messages.tsx:73]: Possible yes, but I'd rather solve this automatically on page load but I currently don't know a way to reproduce this cached reopen where the site state is kept.

Comment (iarp) [reply ui/src/message/Messages.tsx:73]: Assuming your browser reopens previously opened tabs, open gotify in a tab and close the browser. Reopen the browser and thats it. Or open gotify, leave the browser open and sleep the computer.

Comment (jmattheis) [reply ui/src/message/Messages.tsx:73]: I'm doing this, but my firefox correctly reestablishes the websocket connection and has messages that were sent while the browser was closed.

**Pull request merged**